specs/networking/p2p-interface.md

```diff
@@ -765,6 +773,14 @@ Some examples of where messages could be duplicated:
 * Attestation aggregation strategies where clients partially aggregate attestations and propagate them. Partial aggregates could be duplicated
 * Clients re-publishing seen messages
 
+### Why is there `MAXIMUM_GOSSIP_CLOCK_DISPARITY` when validating slot ranges of messages in gossip subnets?
+
+For some gossip channels (e.g. those for Attestations and BeaconBlocks), there are designated ranges of slots during which particular messages can be sent, limiting messages gossiped to those that can be reasonably used in the consensus at the current time/slot. This is to reduce optionality in DoS attacks.
+
+`MAXIMUM_GOSSIP_CLOCK_DISPARITY` provides some wiggle room in validating slot ranges to prevent the gossip network from becoming overly brittle with respect to clock disparity. Although messages can at times be eagerly gossiped to the network, the node's fork choice prevents integration of these messages into the actual consensus until the _actual local start_ of the designated slot.
```

Comment:
```suggestion
`MAXIMUM_GOSSIP_CLOCK_DISPARITY` provides some leeway in validating slot ranges to prevent the gossip network from becoming overly brittle with respect to clock disparity. Although messages can at times be eagerly gossiped to the network, the node's fork choice prevents integration of these messages into the actual consensus until the _actual local start_ of the designated slot.
```